components/TileItem.vue

```diff
@@ -77,7 +78,19 @@ const weekDay = (date: Date) => {
         >
           <NuxtIcon name="icon.play" class="text-white-1 text-2xl p-2" />
         </button>
-        <TrackMenu :track="item.track" class="ml-auto"></TrackMenu>
+        <TrackMenu
+          :track="item.track"
+          :share-text="t('podcast.action.share')"
+          :share-link="
+            item.shufflePodcastId
+              ? {
+                  name: 'playlist-podcast-id',
```

Comment:
Yeah I was a bit conflicted on this, because these `TileItem`s were using the `TrackMenu`, however they seem to actually be podcasts, not tracks.

Should we perhaps have a seperate `TileMenu` component instead?

<img width="1512" alt="Screenshot 2024-02-10 at 23 03 07" src="https://github.com/bcc-code/bmm-web/assets/9109969/c4544273-4661-49bb-9238-b1f8d55118ad">

Reply:
When are you most likely to use that button? I’d say when you’ve just listened to that episode and want to recommend it to someone else. Then it makes more sense to get a link directly to the track, instead of a list of episodes in the podcast, where the top episode has changed since and you’re not sure if you should listen to the 1st, 2nd or 3rd.

we do it the same in the BMM app.